genie-web/src/main/resources/db/migration/h2/V4_0_0__Genie_4.sql

```diff
@@ -146,6 +146,8 @@ ALTER TABLE `jobs`
   ADD COLUMN `claimed` BOOLEAN NOT NULL DEFAULT FALSE;
 ALTER TABLE `jobs`
   ADD COLUMN `v4` BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE `jobs`
+  ADD COLUMN `requested_archive_location_prefix` VARCHAR(1024) DEFAULT NULL;
```

Comment:
@tgianos does modifying this file in place make it hard to deploy to a stack already running V4?
Or does FlyWay magic take care of it?

Reply:
@mprimi Sumit asked me about this too. For now we are ok because our dev/milestone stacks internally blow away our databases on every deployment and take a clone of test and re-apply the migration script. Once we push a version of 4.0 to test/prod we will no longer be able to modify this file.

Flyway works by hashing the migration scripts (which have a convention file name definition to determine order) and using a database lookup table to see if that migration has already been applied. In this case since we blow away the databases of our development stacks it's fine but as I said once this goes to test which we clone from it will break because the hashes of this file (after modifications) would be different than the `V4_0_0__Genie_4.sql` script that was already applied to the test database acting as the source.

Once we push to test we should no longer modify this file and instead should create something like `V4_0_0_1_further_mods.sql` for subsequent modifications and up that 4th digit till official OSS 4.0 release.

It's one reason I wanted to hold off on pushing to test/prod but it shouldn't be too bad. Just more small migration script files.

Let me know if this doesn't make sense @mprimi @sumitnetflix